packages/app-desktop/gui/MainScreen/MainScreen.tsx

```diff
@@ -577,6 +580,14 @@ class MainScreenComponent extends React.Component<Props, State> {
 			await restart();
 		};
 
+		const onCheckForUpdates = async () => {
```

Comment:
This function is duplicated in the `EncryptionConfigScreen.tsx`. Even though it's just a wrapper around `checkForUpdates`, it might make sense to move this into a separate file (or even get rid of the "do you want to update" section of `EncryptionConfigScreen.tsx`).